packages/rs-dpp/src/identity/state_transition/asset_lock_proof/instant/instant_asset_lock_proof.rs

```diff
@@ -102,23 +102,28 @@ impl InstantAssetLockProof {
     }
 
     pub fn out_point(&self) -> Option<[u8; 36]> {
-        self.transaction.out_point_buffer(self.output_index())
+        let out_point_buffer = self.transaction.out_point_buffer(self.output_index());
+
+        // Change tx_id endianness to match js-dpp
+        if let Some(mut out_point_buffer) = out_point_buffer {
```

Comment:
Can we just use .map here? `out_point_buffer.map(|v| {  let (tx_id, _) = v.split_at_mut(32); tx_id.reverse(); tx_id })`

Reply:
Fixed